src/components/Events/EventInfo.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+
+import  EventCard  from './EventCard'
+import Card from '../UI/Card';
+import './EventInfo.css';
+
+const apiUrl = process.env.REACT_APP_SERVER_URL
+
+const DisplayEventInfo = () => {
+    const [ eventData, setEventData ] = useState([]);
+    
+    const getPlayersDataHandler = async () => {
+        try {
+            const result = await fetch(
+                `${apiUrl}/events`,
+            );
+            let responseData = await result.json();
+            responseData = responseData.map(item => {
+                return {
+                    ...item
+                };
+            })
+            if (result.status === 200) {
+                console.log('success fechted', responseData)
+                setEventData(responseData);
+            };
+        } catch (error) {
+            console.log(error);
+            // window.location = "/500"
```

Comment:
maybe we can show an alert, instead? Failing to get events is not a serious problem to use the website, so taking the user to the `500` page might not be a little bit too much, imo.